### PR TITLE
[native] Advance Velox

### PR DESCRIPTION
**Pull request by wypb**
Remove registration of HDFS file-system. This component is now registered in Velox 
during the Hive connector registration.
Remove `presto-native-execution/presto_cpp/main/connectors` code.

### CODE REVIEW

**Comment by wypb** [comment]
Hi @majetideepak @mbasmanova, could you help me review this, thanks.

File: presto-native-execution/presto_cpp/main/CMakeLists.txt [hunk 1]
```diff
@@ -32,6 +31,15 @@ add_library(
   TaskManager.cpp
   TaskResource.cpp)
 
+add_library(presto_adapters INTERFACE)
```
Comment (majetideepak) [inline presto-native-execution/presto_cpp/main/CMakeLists.txt:34]: do we need the presto_adapters library?
velox_hive_connector library is sufficient?

Comment (wypb) [reply presto-native-execution/presto_cpp/main/CMakeLists.txt]: @majetideepak You are right, I have tested it in the local environment, and removing these also works

**Review by majetideepak — COMMENTED** [review]
@wypb thanks for this change.
Can you update the commit message with the description in the PR?

**Comment by wypb** [comment]
Hi @majetideepak Deepak, I modified it according to your suggestion, thanks.

**Review by majetideepak — APPROVED** [review]

**Pull request merged**